pages/feed/index.tsx

```diff
@@ -281,7 +284,8 @@ const Feeds: WithPageLayout<HighlightSSRProps> = (props: HighlightSSRProps) => {
             <FollowingHighlightWrapper selectedFilter={selectedRepo} emojis={emojis} />
           </TabsContent>
         </Tabs>
-        <div className="md:hidden gap-6 mt-10 md:flex-1 lg:flex md:flex-col">
+        
+        <div className="hidden lg:w-[24%] w-[30%] gap-6 mt-10 flex-none lg:flex flex-col">
```

Comment:
@RitaDee  it is not a good practice to use `%` we can't use `w-1/3` -> width: `33.33...%`  `w-4` -> `25%`????

Reply:
@NsdHSO noted with thanks. I will fix that.